CONTRIBUTING.md

```diff
@@ -84,8 +84,17 @@ cd packages/ember-toucan-core
 pnpm start
 # ember-toucan-core is now in watch mode and will rebuild automatically
 
-# In a second terminal
-cd docs-app
+
+# In a second terminal, cd to the root of the repo
+cd root-of-repo-where-that-is-for-you
+pnpm docs:start # using turbo here
```

Comment:
Alright, got farther this time!  Now, for this command, I'm getting:

```
$ pnpm docs:start
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command failed with ENOENT: docs:start
spawn docs:start ENOENT
```

Reply:
Not sure what that error message is about (just checked with @ynotdraw) but the command is here: https://github.com/CrowdStrike/ember-toucan-core/blob/main/package.json#L16

Reply:
Do you have latest `main`?  `[ENOENT]` === `No such file or directory` which makes me think it's not picking up a directory or something potentially?

Running `ls` to see where you're at when you're running this command may help too.  Should be in the root `ember-toucan-core` directory.

Reply:
```suggestion
pnpm start:docs # using turbo here
```

😉